packages/plugin-lighthouse/src/lib/utils.unit.test.ts

```diff
@@ -54,3 +59,272 @@ describe('validateOnlyAudits', () => {
+                endTime: 106_245.559_225,
+                responseReceivedTime: 106_245.559_001,
+                transferSize: 849,
+              },
+            },
+          },
+          longestChain: {
+            duration: 508.498_000_010_848_05,
+            length: 1,
+            transferSize: 849,
+          },
+        },
+      },
+    });
+
+    expect(outputs[0]?.details?.issues[0]).toEqual(
+      expect.objectContaining({
+        message:
+          'Parsing details from type criticalrequestchain is not implemented.',
+        severity: 'info',
+      }),
+    );
+  });
+});
+
+describe('tableToDetails', () => {
+  it('should parse empty lhr details table', () => {
+    expect(
+      tableToDetails({
+        type: 'table',
+        headings: [],
+        items: [],
+      } satisfies Details.Table),
+    ).toStrictEqual({
+      issues: [
+        {
+          message: 'no data present',
+          severity: 'info',
+        } satisfies Issue,
+      ],
+    });
+  });
+
+  it('should parse lhr details table with data', () => {
+    expect(
+      tableToDetails({
+        type: 'table',
+        headings: [
+          {
+            key: 'sourceLocation',
+            valueType: 'source-location',
+            label: 'Source',
+          },
+          {
+            key: 'description',
+            valueType: 'code',
+            label: 'Description',
+          },
+        ],
+        items: [
+          {
+            source: 'network',
+            description:
+              'Failed to load resource: the server responded with a status of 404 ()',
+            sourceLocation: {
+              type: 'source-location',
+              url: 'https://example.com/favicon.ico',
+              urlProvider: 'network',
+              line: 0,
+              column: 0,
+            },
+          },
+        ],
+      } satisfies Details.Table),
+    ).toStrictEqual({
+      issues: [
+        {
+          message: 'sourceLocation, description',
+          severity: 'info',
+        } satisfies Issue,
+      ],
+    });
+  });
+});
+
+describe('opportunityToDetails', () => {
+  it('should parse a valid non-empty lhr opportunity type', () => {
+    expect(
+      opportunityToDetails({
+        type: 'opportunity',
+        headings: [
+          {
+            key: 'url',
+            valueType: 'url',
+            label: 'URL',
+          },
+          {
+            key: 'responseTime',
+            valueType: 'timespanMs',
+            label: 'Time Spent',
+          },
+        ],
+        items: [
+          {
+            url: 'https://staging.code-pushup.dev/login',
+            responseTime: 449.292_000_000_000_03,
+          },
+        ],
+        overallSavingsMs: 349.292_000_000_000_03,
+      } satisfies Details.Opportunity),
+    ).toStrictEqual({
+      issues: [
+        {
+          message: 'url, responseTime',
+          severity: 'info',
+        } satisfies Issue,
+      ],
+    });
+  });
+});
```

Comment:
Yes, this test case actually uses some realistic looking data. This enables us to evaluate the behaviour meaningfully.

Look at the input data and look at the output data. Would you say we converted the Lighthouse data into something useful? The Lighthouse data tells me that what the response time was for a specific URL. But the resulting issue tells me none of this useful information. Instead it cryptically tells me what heading keys are. How is that useful in any way? :confounded: 

However, it seems like the table and opportunities data structures could map to our issues array fairly well, if you want to go down that route.
- A table with static columns can be seen as a fancy list. So we could create an issue for each row (`items` array).
- Our issue's don't support arbitray object data, but a string concatenation of the key=value pairs could serve as the `message`. E.g. to generate something like "`URL: "https://staging.code-pushup.dev/login", Time Spent: 449.292_000_000_000_03`":
  ```ts
  items.map((item): Issue => ({ 
    message: headings.map(({ key, label }) => `${label}: ${JSON.stringify(item[key])}`,
    severity: 'info',
  })
  ```

But the best solution is to refine an issue for extending the `AuditDetails` model to support tabular data.

Reply:
Yes, I opened the PR while you where gone and was hoping to get more attention on the tabular data issue.

I did the hack above to have "something" working besides "not implemented" messages for all detail data.

If you like I can turn all `tables` into issues, or I remove the implementation of detail data parsing for now and add it back later when we have tabular data supported.

Reply:
I'm fine with either, but leaning towards skipping audit details for now.

Reply:
Done 

also see #530